charts/fairspace/templates/project/stateful-set.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       containers:
         - name: {{ .Release.Name }}-postgres-database
           image: postgres:15-alpine
+          args: ["postgres", "-c", "checkpoint_timeout=43200"]
```

Comment:
Did you test it? I am not sure if this would work... 
In k8s configuration file you have a `command` field to define a command and `args` to define arguments for the command and here you use args to define both...  
So I think it should be `args: ["-c", "checkpoint_timeout=43200"]` or something like this

Reply:
Yes I tested :)

Actually the 'postgres' command is not needed, because it is the default value see https://github.com/docker-library/postgres/blob/1a73ab671b5f0f18313726e734c76bf171385c32/15/bookworm/Dockerfile#L219

I will remove it